library/Makefile

```diff
@@ -154,7 +153,10 @@ libmbedx509.dll: $(OBJS_X509) libmbedcrypto.dll
 	echo "  LD    $@"
 	$(CC) -shared -Wl,-soname,$@ -Wl,--out-implib,$@.a -o $@ $(OBJS_X509) -lws2_32 -lwinmm -lgdi32 -L. -lmbedcrypto -static-libgcc $(LOCAL_LDFLAGS) $(LDFLAGS)
 
-libmbedcrypto.%:
+libmbedcrypto.a:
+	$(MAKE) CRYPTO_INCLUDES:="-I../../include -I../include" -C ../crypto/library $@
+
+libmbedcrypto.$(DLEXT): | libmbedcrypto.a
```

Comment:
There is no longer a rule for `libmbedcrypto.so` when `$(DLEXT)` is not `so`, but it's a dependency of `libmbedx509.so.1` and `libmbedtls.so.3`. This breaks the build whenever `DLEXT` is not `so`: macOS (which we don't test on CI, please test manually) and Windows (CI failed on mingw).